site/docs/components/overlay/usage.mdx

```diff
@@ -35,22 +35,25 @@ Interaction with the content in an overlay is typically constrained, as richer i
 To import `Overlay` and related components from the lab Salt package, use:
 
 ```js
-import { Overlay, OverlayTrigger, OverlayPanel } from "@salt-ds/lab";
+import {
+  Overlay,
+  OverlayTrigger,
+  OverlayPanel,
+  OverlayPanelContent,
+  OverlayPanelCloseButton,
+} from "@salt-ds/lab";
 ```
 
 ## Props
 
-{/* Update packageName and componentName below as needed */}
-{/* packageName is optional and defaults to "core" if omitted */}
-
 ### `Overlay`
 
 <PropsTable packageName="lab" componentName="Overlay" />
 
-### `OverlayTrigger`
```

Comment:
Overlay Trigger clones the elements the user adds so doesn't have any props - removed because i didn't think it was useful